lib/database/adapters/QCFlagTypeAdapter.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+/**
+ * QCFlagTypeAdapter
+ */
+class QCFlagTypeAdapter {
```

Comment:
Once again I did not noticed, but for acronyms only first letter is upercase
QcFlagTypeAdapter (same goes for all the rest)

Reply:
applied